apps/etterlatte-saksbehandling-ui/client/src/components/behandling/soeknadsoversikt/styled.tsx

```diff
@@ -20,10 +20,11 @@ export const Vurdering = styled.div`
   padding: 0 0 0 var(--a-spacing-3);
   min-width: 18.75rem;
   width: 10rem;
-  border-left: 3px solid var(--a-border-subtle);
+  border-left: 2px solid var(--a-border-subtle);
```

Comment:
Fins det tokens for border-størrelse? :D

Reply:
Kun for selve fargen, ikke bredde desverre